chemprop/nn/transforms.py

```diff
@@ -41,23 +45,20 @@ def forward(self, X: Tensor) -> Tensor:
 
 
 class GraphTransform(nn.Module):
-    def __init__(self, mean: ArrayLike, scale: ArrayLike, key: str, inds: ArrayLike):
+    def __init__(self, V_mean: ArrayLike, V_scale: ArrayLike, atom_fdim: int, E_mean: ArrayLike, E_scale: ArrayLike, bond_fdim: int):
         super().__init__()
 
```

Comment:
why not directly accept in two `ScaleTransform`s like so:
```suggestion
    def __init__(self, V_transform: ScaleTransform, E_transform: ScaleTransform):
        super().__init__()
        ...
```